timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -361,7 +362,10 @@ def predict(
                     )
 
         # Make sure the item_ids are sorted in the same order as in data
-        return df.loc[data.item_ids]
+        df = df.loc[data.item_ids]
+        # Set correct timestamp index
+        df.index = get_forecast_horizon_index_ts_dataframe(data, self.prediction_length)
+        return df
```

Comment:
A very ad-hoc fix to the GluonTS bug. Should we look for something cleaner?

Reply:
will it work with ignore index?